dockerfiles/examples/ml/scikit/Dockerfile

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+#
+#    HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+
+# Create base container image to be used by all HPCC processes
+# MORE - some of these dependencies are probably not needed by all derived containers - perhaps we should move them
+# Others may not be wanted at all in container mode - scikit example??
+
+ARG BUILD_LABEL
+FROM hpccsystems/platform-core:${BUILD_LABEL}
+USER root
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt clean && \
+    apt autoclean && \
+    apt install -f && \
+    apt autoremove && \
+    apt-get update
```

Comment:
is this needed (platform-core already does this).

Reply:
Just in chance base image apt install need clean-up.
Prefer to keep them. But can be removed also.

Reply:
>Just in chance base image apt install need clean-up.

by definition it's clean I think, so don't know why it would need autoclean , install -f , autoremove, update.

I think should remove, unless reason is clear why needed.

Reply:
You don't need them if you are in lock-step with the core - i.e. this is built every time the core is built.

But if you are building something based on a core built some time back, you might.